pkg/api/operators/apis/experiment/v1alpha2/experiment_types.go

```diff
@@ -50,6 +50,9 @@ type ExperimentSpec struct {
 	// TODO - figure out what to do with metric collectors
 	MetricsCollectorType string `json:"metricsCollectorSpec,omitempty"`
 
+	// For v1alpha2 we will keep the metrics collector implementation same as v1alpha1. 
+	MetricsCollectorSpec *MetricsCollectorSpec `json:"metricsCollectorSpec,omitempty"`
```

Comment:
how about use TrialTemplate to replace MetricsCollectorSpec? or we can rename TrialTemplate to a general name so that MetricsCollectorSpec and TrialTemplate can be same struct.
after that, we can handle both MetricsCollectorSpec and TrialTemplate in same method

Reply:
This is a temporary fix so I think we should keep the definitions separate and more readable.

Reply:
ok